nixos/modules/services/audio/mpd.nix

```diff
@@ -166,6 +166,7 @@ in {
       preStart = ''
         mkdir -p "${cfg.dataDir}" && chown -R ${cfg.user}:${cfg.group} "${cfg.dataDir}"
         mkdir -p "${cfg.playlistDirectory}" && chown -R ${cfg.user}:${cfg.group} "${cfg.playlistDirectory}"
+        mkdir -p "${cfg.musicDirectory}" && chown -R ${cfg.user}:${cfg.group} "${cfg.musicDirectory}"
```

Comment:
Not sure if this is really what we want this because mpd doesn't touch files in the music directory in contrast to the data and playlist directories. For instance, in my case, the music directory is managed by other services which need to drop music in there.

Reply:
This would also break my configuration.  I have the music directory on a samba share and I don't want the owner to be changed to mpd (besides, chown would just fail with permission denied).